src/site/stages/build/process-cms-exports/transformers/page.js

```diff
@@ -0,0 +1,59 @@
+const { flatten, isEmpty } = require('lodash');
+const { getDrupalValue } = require('./helpers');
+
+function createMetaTag(type, key, value) {
+  return {
+    type,
+    key,
+    value,
+  };
+}
+
+function pageTransform(entity) {
+  const transformed = entity;
```

Comment:
If you're trying to not modify `entity`, this won't quite get you there. You'd need `cloneDeep` or something like it.

I think you should keep it like you've got it right now, though, since `cloneDeep` will slow things down a bit.

Just thought I'd mention it is all.

Reply:
Yeah, this _does_ look a bit weird.  I know that the variables are essentially the same - I think initially I meant `transformed` to be more of an indicator of "this is changing" rather than any sort of true separation of objects.